src/utils/getLowerCaseString.ts

```diff
@@ -0,0 +1,7 @@
+export const getLowerCaseString = (value?: string) => {
+    if (!value) {
```

Comment:
you need to do checks out of this function. this simple function should always return a string in a lower case